sdk/nodejs/tests/provider/experimental/testdata/array-types/index.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025-2025, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
+
+export interface MyComponentArgs {
+    anArrayOfStrings: Array<string>;
```

Comment:
Note that for Maps we decided to not handle the `Map` and `Record` type.  For `Array`s though, internally they seem to be the same in the index signature form and this form, so I think it would be harder to support one but not the other here.